src/lib/y2storage/encryption_processes/secure_key.rb

```diff
@@ -107,8 +107,8 @@ class << self
         #
         # @return [Boolean]
         def available?
-          # TODO: likely implemented by calling lszcrypt
-          raise NotImplementedError
+          device_list = Yast::Execute.locally("lszcrypt", "--verbose", stdout: :capture)
```

Comment:
I'm asking because the `Yast::Execute.locally` documentation says: "Shows a {ReportClass#Error popup} if the command fails and returns `nil` in such case."

Reply:
I've tried it, and saw `nil`. Also, `locally!` crashes when the command is not there. I believe the doc is a bit outdated. Just saying...

Reply:
Uhm, I see. `locally!` is expect to raise and exception.